airbyte-webapp/src/components/connectorBuilder/useManifestToBuilderForm.test.ts

```diff
@@ -57,7 +56,7 @@ describe("Conversion throws error when", () => {
       throw new Error(errorMessage);
     };
     const convert = async () => {
-      return convertToBuilderFormValues(resolve, {} as ConnectorManifest, DEFAULT_BUILDER_FORM_VALUES);
+      return convertToBuilderFormValues(resolve, {} as ConnectorManifest, "Untitled");
```

Comment:
Could we put `"Untitled"` into a constant somewhere and reference that constant in all of these tests instead of repeating the string? This will make it easier to change in the future if we want